app/src/main/java/com/droidhats/campuscompass/adapters/ExplorePlaceAdapter.kt

```diff
@@ -0,0 +1,62 @@
+package com.droidhats.campuscompass.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.cardview.widget.CardView
+import androidx.recyclerview.widget.RecyclerView
+import com.droidhats.campuscompass.R
+import com.droidhats.campuscompass.views.ExploreCategoryFragment
+import kotlinx.android.synthetic.main.explore_recycler_item.view.*
+import com.droidhats.campuscompass.models.Explore_Place
+import com.squareup.picasso.Picasso;
+
+
```

Comment:
I would suggest that we add short Java doc comments to our newly introduced classes.

Reply:
Fixed